app/helpers/articles_helper.rb

```diff
@@ -54,4 +54,8 @@ def get_host_without_www(url)
     host.gsub!("medium.com", "Medium")
     host.delete_prefix("www.")
   end
+
+  def formated_date(timestamp)
```

Comment:
missing `t`

I would also call this `utc_iso_timestamp` or something more explicit :D